brig/cmd/brig/commands/run.go

```diff
@@ -54,7 +65,8 @@ the worker successfully ran to completion.
 func init() {
 	run.Flags().StringVarP(&runFile, "file", "f", "", "The JavaScript file to execute")
 	run.Flags().StringVarP(&runEvent, "event", "e", "exec", "The name of the event to fire")
-	run.Flags().StringVarP(&runPayload, "payload", "p", "", "The path to a payload file")
+	run.Flags().StringVarP(&runPayloadFile, "payload-file", "p", "", "The path to a payload file")
```

Comment:
This appears to be a breaking change. The longer form of the `-p` flag should remain `payload` to avoid this breakage.

Reply:
Once this is fixed, the PR will LGTM.

Reply:
Does it look good?

Reply:
Updated!